seldon/seldon-core-operator/base/resources.yaml

```diff
@@ -3515,14 +3515,6 @@ webhooks:
       path: /mutate-machinelearning-seldon-io-v1-seldondeployment
   failurePolicy: Fail
   name: mseldondeployment.kb.io
-  namespaceSelector:
```

Comment:
Should we keep the namespaceSelectors? I think those are supported in K8s 1.14.
Also should we change the namespace selector to only match the profile created namespaces.

See: kubeflow/kfserving#568 I think we want to scope it to namespaces where models should be deployed.

See: https://github.com/kubeflow/kubeflow/blob/c80316d3a646808da4e59ead71ddd1e95d1fb796/components/profile-controller/controllers/profile_controller.go#L65

We automatically add the label "serving.kubeflow.org/inferenceservice": "enabled" to profile created namespaces.
So I think it make sense to scope seldon webhooks using that label as well. This ensures problems with the seldon webhooks don't affect other namespaces .